src/Fixer/PhpBasic/CodeStyle/SplittingInSeveralLinesFixer.php

```diff
@@ -110,6 +110,25 @@ protected function applyFix(SplFileInfo $file, Tokens $tokens): void
         do {
             foreach ($startEndTokens as $startValue => $endValue) {
                 if ($token->getContent() === $startValue) {
+                    // closure situation
+                    if ($token->getContent() === '(') {
```

Comment:
Would be better to use private method that would return correct $token or null

Reply:
Fixed